common/runtime/src/main/java/io/quarkiverse/googlecloudservices/common/GcpConfiguration.java

```diff
@@ -19,4 +19,16 @@
      */
     @ConfigItem
     public Optional<String> serviceAccountLocation;
+
+    /**
+     * Enable Google Cloud access token authentication
+     * For example, the access token which is returned as part of OpenId Connect Authorization Code Flow
+     * may be used to access Google Cloud services on behalf of the authenticated user.
+     *
+     * Note that if a service account location is configured then the access token will be ignored even if this property is enabled.
+     *
+     * Disable this property if the default Google Cloud authentication is required.
+     */
+    @ConfigItem(defaultValue= "true")
+    public boolean accessTokenEnabled = true;
```

Comment:
Don't assign it to true as you already use `defaultValue`

Reply:
@loicmathieu Sure, it is just in Quarkus OIDC this is the pattern we follow to support when the users creating `OidcTenantConfig` manually in their providers...But yeah, here I'll just remove `true`